### PR TITLE
RAGHU S 🍉

### PR DESCRIPTION
**Pull request by raghu-007**
Adding my name to contributor list!!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <br />
                     <sub><b>Harsh Pandey</b></sub>
                 </a>
+            </td>
+             <td align="center">
+                <a href="https://github.com/raghu-007">
+                    <img src="https://avatars.githubusercontent.com/u/50697306?v=4" width="100px;" alt="Harsh Pandey"/>
```
Comment (alisolanki) [inline README.md:57]: Please add your own name in alt

File: README.md [hunk 2]
```diff
@@ -51,6 +51,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <br />
                     <sub><b>Harsh Pandey</b></sub>
                 </a>
+            </td>
+             <td align="center">
```
Comment (alisolanki) [inline README.md:55]: Please remove extra indentation

**Review by alisolanki — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raghu-007) [reply README.md]: Done

**Comment by raghu-007** [comment]
@alisolanki plz do merge my PR

**Pull request merged**